runtime/errors.go

```diff
@@ -158,3 +158,20 @@ func (e *InvalidTypeAssignmentError) Unwrap() error {
 func (e *InvalidTypeAssignmentError) Error() string {
 	return fmt.Sprintf("cannot assign type %s to %s", e.Type, e.Value)
 }
+
+// ScriptReturnTypeNotStorableError is an error that is reported for
+// script return types that are not storable.
+//
+// For example, the type `Int` is a storable type,
+// whereas a function type is not.
+
+type ScriptReturnTypeNotStorableError struct {
+	Type sema.Type
+}
+
+func (e *ScriptReturnTypeNotStorableError) Error() string {
+	return fmt.Sprintf(
+		"return type is non-storable type: %s",
+		e.Type,
```

Comment:
In the error messages we should show the fully-qualified type identifiers to avoid confusion, an we quote them in backticks.

```suggestion
		"return type is non-storable type: `%s`",
		e.Type.QualifiedString(),
```

We already do this everywhere in the sema errors, but not yet here.

While at it, could you please fix the other runtime errors the same way:

- https://github.com/onflow/cadence/blob/benjaminkvm/186-reject-non-storable-return-type/runtime/errors.go#L159
- https://github.com/onflow/cadence/blob/benjaminkvm/186-reject-non-storable-return-type/runtime/errors.go#L174-L175
- https://github.com/onflow/cadence/blob/benjaminkvm/186-reject-non-storable-return-type/runtime/errors.go#L98

Reply:
Sure thing!